api/app/src/main/kotlin/packit/model/Packet.kt

```diff
@@ -0,0 +1,25 @@
+package packit.model
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import jakarta.persistence.*
+
+@Entity
+@Table(name = "packet")
+data class Packet(
+    @Id
+    val id: String,
+    val name: String,
+    val displayName: String,
+    @Column(name = "parameters", columnDefinition = "json")
+    val parameters: String,
+    val published: Boolean,
```

Comment:
I have restored val for simplicity. I think we should observe the immutability but this seems to work for now.